### PR TITLE
fix(hosting) allow cloudweb cdn perf as free option as in business model

### PR DESCRIPTION
**Pull request by VincentCasse**
### Description of the Change

CloudWeb offers include free cdn option.
Inside customer panel, we sell it.

MBP-374

### Possible Drawbacks

Can break the CDN option sell

### CODE REVIEW

File: client/app/hosting/cdn/order/hosting-cdn-order.controller.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ angular
           }
           $scope.loading.model = true;
           HostingOptionOrder.getOrderEnums('hosting.web.CdnOfferEnum').then((models) => {
-            if (!$scope.isPerfOffer()) {
+            if (!$scope.isPerfOffer() and !$scope.isCloudWebOffer() ) {
```
Comment (frenautvh) [inline client/app/hosting/cdn/order/hosting-cdn-order.controller.js]: missing offer parameter

File: client/app/hosting/hosting.service.js [hunk 2]
```diff
@@ -121,6 +121,15 @@
         return /^perf.+$/.test(offer);
       }
 
+      /**
+             * Is hosting cloudweb offer
```
Comment (marie-j) [inline client/app/hosting/hosting.service.js]: The indentation seems a bit off here

File: client/app/hosting/hosting.service.js [hunk 3]
```diff
@@ -121,6 +121,15 @@
...
+             * @param {string} offer
+             * @returns {boolean}
+             */
+      static isCloudWebOffer(offer) {
+        return _(offer).includes('CLOUD');
```
Comment (marie-j) [inline client/app/hosting/hosting.service.js]: Prefer using `_.includes(offer, 'CLOUD')` instead

**Comment by euhmeuh** [comment]
This should be enough: https://github.com/ovh-ux/ovh-manager-web/pull/1130

**Pull request closed without merging**